resource/specs/staging.toml

```diff
@@ -57,6 +57,17 @@ lock.code_hash = "0x54811ce986d5c3e57eaafab22cdd080e32209e39590e204a99b32935f835
 lock.args = ["0x57ccb07be6875f61d93636b0ee11b675494627d2"]
 lock.hash_type = "Data"
 
+# Dep group cells
```

Comment:
Put it after `[genesis.system_cells.lock]`.

Reply:
Fine with me. My original thought is dep groups are in 2nd transaction.